scripts/zones/Port_Jeuno/npcs/Sagheera.lua

```diff
@@ -414,4 +497,30 @@ entity.onEventFinish = function(player, csid, option)
     end
 end
 
+local handleTradeChipEvent = function(player, option)
+    local trade = player:getTrade()
+    if (npcUtil.tradeSetInList(trade, tier1Chips) and npcUtil.giveItem(player, { { xi.items.ANCIENT_BEASTCOIN, tier1ChipValue } })) then
```

Comment:
No need for outer parens here

Reply:
Whoops, I was starting to merge these two conditions with an or but they're so lengthy ; ;

Reply:
For length, might want to consider our multi-line style as well:
```
if
    condition1 and
    not condition2
then

end
```